Source/Csla.Blazor.WebAssembly/State/SessionManager.cs

```diff
@@ -40,35 +40,44 @@ public class SessionManager(
     /// </summary>
     public Session GetSession()
     {
-      _session ??= [];
-      _session.LastTouched = DateTimeOffset.UtcNow;
+      if (_session == null)
+      {
+        _session ??= [];
+        _session.LastTouched = DateTimeOffset.UtcNow.ToUnixTimeSeconds();
+      }
       return _session;
     }
 
     /// <summary>
     /// Updates the current user's session 
-    /// data locally and on the web server.
+    /// data locally.
     /// </summary>
     /// <param name="session">Current user session data</param>
     public void UpdateSession(Session session)
     {
-      _session.LastTouched = DateTimeOffset.UtcNow;
+      _session.LastTouched = DateTimeOffset.UtcNow.ToUnixTimeSeconds();
```

Comment:
You are updating here the old session. Seems like a bug? Maybe you could extract the method I proposed in SessionManager with an overload to unify this kind of updating the `LastTouched` field.